src/modules/mavlink/module.yaml

```diff
@@ -95,3 +104,45 @@ parameters:
             reboot_required: true
             num_instances: *max_num_config_instances
             default: [true, true, true]
+
+        MAV_${i}_UDP_PORT:
+            description:
+                short: MAVLink Network Port for instance ${i}
+                long: |
+                    If ethernet enabled and selected as configuration for MAVLink instance ${i}, 
+                    selected udp port will be set and used in MAVLink instance ${i}.
+
+            type: int32
+            reboot_required: true
+            num_instances: *max_num_config_instances
+            default: [14556, 0, 0]
+            ethernet:
+
+        MAV_${i}_REMOT_PORT:
```

Comment:
REMOT or REMOTE?

Reply:
@hamishwillee REMOT... REMOTE will bring us to 17 chars and 16 is the limit

Reply:
Thanks @garfieldG I guess I should have counted. FWIW I probably would have done REMOTE_PRT, although I guess a matter of taste

Reply:
I agree with @hamishwillee REMOTE_PRT looks better

Reply:
@hamishwillee Changed it. Also changed UDP_PORT to UDP_PRT to be similar

Reply:
Thanks @garfieldG .I wouldn't have suggested you do that (even for consistency) but it is completely fine :-).